libmamba/tests/src/core/test_util_os.cpp

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) 2019, QuantStack and Mamba Contributors
```

Comment:
Same here.
And btw, should we put this test file in `tests/src/util` folder instead of `core`? But I guess `test_system_env` could be kept in core?

Reply:
I would move the tests file at the same time as we are able to move the header under similar `mamba/util` `mamba::util` scope.

Reply:
Yeah I tried to match where the current header is, not all utils are moved in `mamba/util/` at the moment.